test/tests/outputs/json/one_file_per_timestep/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  issues = '#18133'
+  design = 'JSONOutput.md'
+  [basic]
+    type = CheckFiles
```

Comment:
You have gold files, but you are only checking that the files exist after running. Are you wanting to do a JSON diff? If not, I would recommend removing the gold files.

Reply:
Good catch, these should be a proper CSVDiff